docs/caches.md

```diff
@@ -43,4 +43,4 @@ AppCache doesn't support conditional or delayed assets loading and by default ig
  AppCache: {
   caches: ['main', 'additional', 'optional']
  }
-```
\ No newline at end of file
+```
```

Comment:
oh, I see. I don't have any special feeling about trailing like breaks, I just don't like unnecessary/unrelated changes because who knows when you will need to merge things and why they could break then.